### DISCUSSION TITLE
Ways to add some content after each list?

### DISCUSSION THREAD

**Discussion by guyutongxue** [typst/typst#1114] [Q&A]
I want to add some content for each `list` in my document. Is there any way (`show` rules or something else) to do that?

Specifically, I want to add an empty paragraph after every list to resolve #311 `first-line-indent` problem.

**Comment by yochem** [comment]
I think this is what you want:

```typst
#set par(first-line-indent: 2em)
#show list: l => {
  l
  par(text(size:0.35em, h(0.0em)))
}

This is not indented
- x
- y

This is also indented

This is indented
```

Gives:

![image](https://github.com/typst/typst/assets/23235841/0e7daddc-318c-4142-8b37-fdd594e60889)

**Reply by guyutongxue** [reply]
Well... I totally forgot that I've asked this question :joy:

Anyway, my initial thought is to keep first-line-indent on every paragraph, by typing:

```typst
A paragraph
- Item 1
- Item 2

Next paragraph
```

Should got something like:

```
        A paragraph

  o   Item 1
  o   Item 2

        Next paragraph
```

By the time I ask this question (with Typst 0.3 or 0.4), seems that no settings or `#show` rules can produce this result.

I'm now just waiting for some improvement on #311. Thanks for your reply :eyes:
